src/software/amazon/ion/impl/lite/IonSymbolLite.java

```diff
@@ -14,17 +14,12 @@
 
 package software.amazon.ion.impl.lite;
 
+import software.amazon.ion.*;
```

Comment:
Let's not introduce style changes into the fix.

Reply:
I always do a squash merge when there are multiple commits, and yes, the `.*` import will be fixed before merge.